src/main/resources/stairway/db/changesets/20200310_scale.yaml

```diff
@@ -0,0 +1,36 @@
+databaseChangeLog:
+  - changeSet:
+      id: stairwaybase
+      author: dd
+      changes:
+        - addColumn:
+            tableName: flight
+            columns:
+              - column:
+                  name: stairway_id
+                  type: text
+                  remarks: identifier for this Stairway instance
+
+        - addColumn:
+            tableName: flightlog
+            columns:
+              - column:
+                  name: rerun
+                  type: boolean
+                  remarks: true if rerun of the current step is requested by the step
+
+        - createTable:
+            tableName: stairwayinstance
+            columns:
+              - column:
+                  name: stairway_name
+                  type: text
+                  remarks: externally set name of the stairway instance
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: stairway_id
```

Comment:
This column is also unique, right? I think adding an explicit "unique" constraint would be helpful.

Reply:
OK